app/assets/javascripts/app/core/models/discussion.js

```diff
@@ -36,6 +36,10 @@
           var discussion = this;
 
           return DS.refresh('discussion', discussion.id);
+        },
```

Comment:
I removed the `linkInverse` but the endpoint is still needed, like I said before, this happens because of the parameters sent to the backend, I cannot send the `discussion_id`, but without it the generated url isn't valid. For now, this is the best option.